cmd/infracost/main.go

```diff
@@ -62,8 +53,8 @@ func main() {
 			},
 			&cli.StringFlag{
 				Name:  "log-level",
-				Usage: "Log level (TRACE, DEBUG, INFO, WARN, ERROR)",
-				Value: "WARN",
+				Usage: "Log level (trace, debug, info, warn, error, fatal)",
```

Comment:
can you possibly update `cmd/infracost-generate-docs/main.go` so it's consistent?

Reply:
:+1: will fix this and other resources to be consistent